assets/sass/components/reader-revenue-manager/_googlesitekit-rrm-setup-module.scss

```
@@ -19,34 +19,29 @@
 .googlesitekit-plugin {
 
 	.googlesitekit-setup-module__publication-create-screen {
-		.googlesitekit-setup-module__title {
-			color: $c-surfaces-on-surface;
+
+		.googlesitekit-setup-module__title,
+		.googlesitekit-setup-module__description {
 			font-family: $f-primary;
-			font-size: $fs-body-lg;
+			font-size: $fs-body-md;
 			font-weight: $fw-normal;
-			letter-spacing: $ls-m;
-			line-height: $lh-body-lg;
+			letter-spacing: $ls-s;
+			line-height: $lh-body-md;
+		}
+
+		.googlesitekit-setup-module__title {
+			color: $c-surfaces-on-surface;
 			margin-bottom: 0;
 		}
 
 		.googlesitekit-setup-module__description {
 			color: $c-surfaces-on-background-variant;
-			font-family: $f-primary;
-			font-size: $fs-body-md;
-			font-weight: $fw-normal;
-			letter-spacing: $ls-s;
-			line-height: $lh-link-md;
 			margin-top: 0;
```

This `margin-top` needs to be ~3px to match the Figma design and improve spacing between the title and description.

I'll apply the change myself to save the round trip.

```suggestion
			margin-top: 3px;
```

---

Thanks, @techanvil.